modules/ROOT/pages/clauses/delete.adoc

```diff
@@ -101,6 +124,33 @@ For more information, see link:{neo4j-docs-base-uri}/operations-manual/{page-ver
 ====
 
 
+[[delete-a-single-node]]
+== Delete single node
+
+To delete a single node that has no relationships connected to it, use the `NODETACH DELETE` clause (or simple `DELETE`).
+
+.Query
+[source, cypher, indent=0]
+----
+MATCH (n:Person {name: 'Carrie-Anne Moss'})
+NODETACH DELETE n
+----
+
+This deletes the `Person` node `Carrie-Anne Moss` and all relationships connected to it.
+
+.Result
+[role="queryresult",options="footer",cols="1*<m"]
+----
+Deleted 1 node, deleted 1 relationship
+----
+
+[NOTE]
+====
+The `DETACH DELETE` clause may not be permitted to users with restricted security privileges.
+For more information, see link:{neo4j-docs-base-uri}/operations-manual/{page-version}/authentication-authorization/access-control#detach-delete-restricted-user[Operations Manual -> Fine-grained access control].
+====
+
+
```

Comment:
This part is slightly confusing for two reasons:
- it creates a new header called "Delete single node" (there is already one with the same name - see line 36)
- it first states "delete a single node that has no relationships connected to it.." and then, after the example, says "This deletes the `Person` node `Carrie-Anne Moss` and all relationships connected to it." (It may not be obvious to readers that Carrie's relationship was deleted in the previous example).

More broadly, I don't think this part needs to be included. We have already mentioned NODETACH above, which probably includes all that is necessary, right?

Reply:
Oh this wasn't meant to be here 🙈 I forgot to remove it